suse_migration_services/defaults.py

```diff
@@ -46,3 +46,11 @@ def get_target_kernel(self):
     @classmethod
     def get_target_initrd(self):
         return 'boot/initrd'
+
+    @classmethod
+    def get_ssh_keys_path(self):
+        return '/home/*/.ssh/authorized_keys'
```

Comment:
This is not sufficient. While the images we publish by default do not have a configuration that would create "/root/.ssh/authorized_keys" we cannot guarantee that a sysadmin of an instance did not create this and set the VM up more like a "traditional" system. Most likely that user/sysadmin would also be the one migrating the system. Meaning we have to check/account for "/root/.ssh/authorized_keys"

Reply:
Good point. That case should be solved now.